libraries/USBHost/examples/ADKTerminalTest/ADKTerminalTest.ino

```diff
@@ -37,7 +37,7 @@ void setup(void)
   while (!SERIAL_PORT_MONITOR); // Wait for serial port to connect - used on Leonardo, Teensy and other boards with built-in USB CDC serial connection
   SERIAL_PORT_MONITOR.println("\r\nADK demo start");
 
-  if (usb.Init() == -1)
+  if (usb.Init() == (uint32_t)-1)
```

Comment:
hmm, it doesn't look right to me,

Reply:
I don't understand.  This casts the value `-1` to `uint32_t`, because that is the return type of the function.  I did not write this example, just stopped the compiler warning?

Reply:
Ah you're right, the author of this sketch is very tricky. I guess casting it is good enough. But that still look weird